receipt/README.md

```diff
@@ -30,7 +30,7 @@ From online shoe stores to indie music platforms, any company selling products o
 Obviously, you should have a [Mailjet](https://www.mailjet.com/) account. If you're not a client yet, you can [subscribe for a free account](https://app.mailjet.com/signup) (and send immediately up to 6000 free emails a month!). __Warning:__ don't use a disposable email address (gmail, yahoo, etc included), as you'll be likely to be blocked by our anti-spammer policy.
 
 Some basic knowledge about [MJML](https://mjml.io/) is a plus, but isn't mandatory. If you prefer to code your email in HTML, we provide you with a ready-to-use `index.html` file that you can find at the root of this repository.
```

Comment:
All your code snippets use 'advanced' features of MJML like mj-class and mj-style but it's never explained what MJML actually is. I think it would be good to at least explain that MJML is a markup language making responsive email easy instead of dealing with messy & unreadable HTML.